sources/academy/webscraping/puppeteer_playwright/common_use_cases/paginating_through_results.md

```diff
@@ -53,7 +53,7 @@ Let's grab this number now with a little bit of code:
 
 ```javascript
 import { chromium } from 'playwright';
-import { load } from 'cheerio';
+import * as cheerio from 'cheerio';
```

Comment:
Same here, maybe I'm missing something though (or maybe it's that we've used cheerio in this example before?)